tools/change_etype_to_canonical_etype.py

```diff
@@ -41,71 +46,81 @@ def convert_conf(part_config):
             f.truncate()
 
 
-def etype2canonical_etype(part_config):
-    gpb, _, _, etypes = load_partition_book(part_config=part_config, part_id=0)
-    eid = []
-    etype_id = []
-    for etype in etypes:
-        type_eid = torch.zeros((1,), dtype=torch.int64)
-        eid.append(gpb.map_to_homo_eid(type_eid, etype))
-        etype_id.append(etypes[etype])
-    eid = torch.cat(eid, 0)
-    etype_id = torch.IntTensor(etype_id)
-    partition_id = gpb.eid2partid(eid)
-    canonical_etypes = {}
-    part_ids = [
-        part_id
-        for part_id in range(gpb.num_partitions())
-        if part_id in partition_id
-    ]
-    for part_id in part_ids:
-        seed_edges = torch.masked_select(eid, partition_id == part_id)
-        seed_edge_tids = torch.masked_select(etype_id, partition_id == part_id)
-        c_etype = _find_c_etypes_in_partition(
-            seed_edges, seed_edge_tids, part_id, part_config
-        )
-        canonical_etypes.update(c_etype)
-    return canonical_etypes
+def etype2canonical_etype(part_config, config):
+    try:
+        num_parts = config["num_parts"]
```

Comment:
how about define such magic strings into variables and use CAPTIAL for global variables?

Reply:
make sense, fix